frontend/src/components/InnerNavbar/index.tsx

```diff
@@ -0,0 +1,18 @@
+/***** CONSTS *****/
+import React from 'react';
+import './InnerNavbar.scss';
+
+/***** TYPE DEFINITIONS *****/
+type TSidebar = React.FC<{
+  children: React.ReactNode;
+}>
+
+export const Sidebar: TSidebar = ({children}) => {
+  return(
+    <div className="SideBar">
+      {React.Children.map(children, (child, index) => (
+        <div key={index}>{child}</div>
+      ))}
```

Comment:
So with children, similar to the other components we have written, you don't need to do anything special with them - this comment is essentially a "wrapper" or "container" around the children passed in and does nothing more really. you can just render them as `{children}`.

 The only time that you would use React.Children.map is when you are wanting to take the children and perform some kind of mapping operation over them. 

The reason that you would want to use this helper function from react, is specifically so that it only gives you valid react children and not ones such as null or false.

In this case, you can simple render children like in the other components
```suggestion
      {children}
```

I'd recommend reading this guide on what the curly brackets actually do within the JSX and why the suggested code here is actually going to render the children (vs writing the word children)
https://react.dev/learn/javascript-in-jsx-with-curly-braces#using-curly-braces-a-window-into-the-javascript-world